.travis.yml

```diff
@@ -10,49 +10,60 @@ branches:
 cache:
   yarn: true
 
-stages:
```

Comment:
What was wrong with the multi stage build?

Reply:
nothing. I just run travis build to find the problem.